src/Serialization/Address.purs

```diff
@@ -168,6 +168,9 @@ showVia nm toShowable addr = "(" <> nm <> " " <> show (toShowable addr) <> ")"
 instance Eq Address where
   eq = eq `on` addressBytes
 
+instance Ord Address where
+  compare a b = addressBytes a `compare` addressBytes b
```

Comment:
Just to follow a common convention in our libraries, I think `compare = compare on addressBytes` would be better, and it's point free.